raiden/smart_contracts/ChannelManagerContract.sol

```diff
@@ -82,6 +82,7 @@ contract ChannelManagerContract {
     /// @param settle_timeout The settle timeout in blocks
     /// @return The address of the newly created NettingChannelContract.
     function newChannel(address partner, uint settle_timeout) returns (address) {
+        require(msg.sender != partner);
```

Comment:
This is not required. It's checked in the `NettingChannelContract` constructor.

https://github.com/raiden-network/raiden/blob/e17d96db375d31b134ae7b4e2ad2c1f905b47857/raiden/smart_contracts/NettingChannelContract.sol#L27